python/CMakeLists.txt

```diff
@@ -6,39 +6,41 @@ cmake_minimum_required (VERSION 2.8)
 #
 ######################################################################
 
-FIND_PACKAGE(PythonInterp 2 REQUIRED)
-
-IF(PYTHONINTERP_FOUND)
-    # check that Python version 2.x is used
-    execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
-                         "import sys; print(sys.version[0])"
-                          OUTPUT_VARIABLE PYTHON_MAJOR_VERSION OUTPUT_STRIP_TRAILING_WHITESPACE)
-    IF(NOT (${PYTHON_MAJOR_VERSION} EQUAL 2))
-        MESSAGE(FATAL_ERROR "Python bindings require Python 2.x.")
-    ENDIF()
-ENDIF()
+FIND_PACKAGE(PythonInterp REQUIRED)
+
+#IF(PYTHONINTERP_FOUND)
+#    # check that Python version 2.x is used
+#    execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
+#                         "import sys; print(sys.version[0])"
+#                          OUTPUT_VARIABLE PYTHON_MAJOR_VERSION OUTPUT_STRIP_TRAILING_WHITESPACE)
+#    IF(NOT (${PYTHON_MAJOR_VERSION} EQUAL 2))
+#        MESSAGE(FATAL_ERROR "Python bindings require Python 2.x.")
+#    ENDIF()
+#ENDIF()
 
 ######################################################################
 #
 #      find Python library
 #
 ######################################################################
 
-execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
-                 "import sys; print sys.exec_prefix"
-                  OUTPUT_VARIABLE PYTHON_PREFIX OUTPUT_STRIP_TRAILING_WHITESPACE)
+find_package(PythonLibs REQUIRED)
 
-IF(APPLE AND ${PYTHON_PREFIX} MATCHES ".*framework.*")
-    SET(PYTHON_LIBRARIES "${PYTHON_PREFIX}/Python"
-        CACHE FILEPATH "Python libraries"
-        FORCE)
-ELSE()
-    execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
-                     "import sys; skip = 2 if sys.platform.startswith('win') else 1; print 'python' + sys.version[0:3:skip]"
-                      OUTPUT_VARIABLE PYTHON_LIBRARY_NAME OUTPUT_STRIP_TRAILING_WHITESPACE)
-    FIND_LIBRARY(PYTHON_LIBRARIES ${PYTHON_LIBRARY_NAME} HINTS "${PYTHON_PREFIX}" 
-                 PATH_SUFFIXES lib lib64 libs DOC "Python libraries")
-ENDIF()
+#execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
+#                 "import sys; print(sys.exec_prefix)"
+#                  OUTPUT_VARIABLE PYTHON_PREFIX OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+#IF(APPLE AND ${PYTHON_PREFIX} MATCHES ".*framework.*")
+#    SET(PYTHON_LIBRARIES "${PYTHON_PREFIX}/Python"
+#        CACHE FILEPATH "Python libraries"
+#        FORCE)
+#ELSE()
+#    execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
+#                     "import sys; skip = 2 if sys.platform.startswith('win') else 1; print('python' + sys.version[0:3:skip])"
+#                      OUTPUT_VARIABLE PYTHON_LIBRARY_NAME OUTPUT_STRIP_TRAILING_WHITESPACE)
+#    FIND_LIBRARY(PYTHON_LIBRARIES ${PYTHON_LIBRARY_NAME} HINTS "${PYTHON_PREFIX}" 
+#                 PATH_SUFFIXES lib lib64 libs DOC "Python libraries")
+#ENDIF()
```

Comment:
Should add that while I haven't built this package before. I have had some rough experiences with CMake finding the right Python interpreter. If this works, great, but it might be worth checking with others who added this to verify it still works.